rpcs3/Emu/System.cpp

```diff
@@ -1346,7 +1459,6 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 				CallFromMainThread([this]
 				{
 					Emu.Kill(false);
-					m_path = m_path_old; // Reset m_path to fix boot from gui
```

Comment:
removing this here makes no sense

Reply:
It's no longer needed because I restore it in BootGame.

Reply:
you do not

Reply:
Added it for this special case.